odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -46,6 +48,8 @@
     'website_sale_default_country': 'website_sale',
     # OCA/event
     'event_registration_mass_mailing': 'mass_mailing_event',
+    # OCA/hr
+    ''
```

Comment:
Is this required?

Reply:
I dont know why this appears like this here, in my file it says 
# OCA/hr
    'hr_employee_seniority': 'hr_employee_service_contract',